crates/sui-core/src/unit_tests/data/transitive_dependencies/package_B_depends_on_C/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "PackageBDependsOnC"
+version = "0.0.1"
+
+[addresses]
```

Comment:
A trick I used for the source validation fixtures was to not include the `addresses` section, and then concatenate it on the end when copying the fixture over, see, e.g. 

https://github.com/MystenLabs/sui/blob/8b64e6c78f572d76e331040e62aac640f570bd83/crates/sui-source-validation/src/tests.rs#L608-L612